iModelCore/ecobjects/PublicApi/ECObjects/ECSchema.h

```diff
@@ -3831,7 +3831,7 @@ friend struct SchemaJsonWriter;
     //! @param[in]  sourceClass The class to copy
     //! @param[in]  copyReferences If true the method will copy types from the source schema into the target schema, if they do not already exist. If false, there will be a schema reference created to the source schema if necessary.
     //! @param[in]  newName  If not nullptr, this name will be used as the new name instead of the original name
-    ECOBJECTS_EXPORT ECObjectsStatus CopyClass(ECClassP& targetClass, ECClassCR sourceClass, bool copyReferences = false, Utf8CP newName = nullptr);
+    ECOBJECTS_EXPORT ECObjectsStatus CopyClass(ECClassP& targetClass, ECClassCR sourceClass, bool copyReferences = false, Utf8CP newName = nullptr, bool skipValidations = false);

```

Comment:
let's call it skipValidation (no plural) to match how it's called in SchemaContext